Jenkinsfile

```
@@ -1,3 +1,23 @@
+// Minor housekeeping logic
+boolean specialBranch = env.BRANCH_NAME.equals("master") || env.BRANCH_NAME.equals("develop")
+
+String[] jobNameParts = env.JOB_NAME.tokenize('/') as String[]
+String realPojrectName = jobNameParts.length < 2 ? env.JOB_NAME : jobNameParts[jobNameParts.length - 2]
```

Minor typo :-) Enough to make build go boom.
```suggestion
String realProjectName = jobNameParts.length < 2 ? env.JOB_NAME : jobNameParts[jobNameParts.length - 2]
```